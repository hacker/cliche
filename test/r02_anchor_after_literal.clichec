#include <iostream>
int main() {
const char *l3 = <%literal>line 3</%literal> ":"; int nl3 = __LINE__;
std::cout << l3 << nl3 << std::endl;
{<%output>line 5 (<% __LINE__ %>)</%output>}
int l6 = __LINE__;
std::cout << std::endl << "line 6: " << l6 << std::endl;
};
