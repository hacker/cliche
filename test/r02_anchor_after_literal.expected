line 3:3
line 5 (5)
line 6: 6
